DatadogTrace/Sources/OpenTelemetry/OTelSpan.swift

```diff
@@ -136,42 +136,20 @@ internal class OTelSpan: OpenTelemetryApi.Span {
         )
     }
 
-    /// Sends a span event which is akin to a log in Datadog
-    /// - Parameter name: name of the event
     func addEvent(name: String) {
-        addEvent(name: name, timestamp: .init())
+        // not supported
```

Comment:
suggestion/ We may consider warning the user of depending on not supported API:
```swift
DD.logger.warn("\(#function) is not yet supported in `DatadogTrace`")
```

Reply:
I was hesitant that it can be too verbose but glad you mentioned.

Updated.